guides/features/pipecat-flows.mdx

```diff
@@ -466,517 +292,273 @@ NodeConfig(
             }
         ],
         "task_messages": [
             {
                 "role": "system",
-                "content": "Ask for the customer's age."
+                "content": "Say 'Hello world' and ask what is the user's favorite color.",
             }
         ],
-        "functions": [
-            {
-                "type": "function",
-                "function": {
-                    "name": "collect_age",
-                    "handler": collect_age,
-                    "description": "Collect customer age",
-                    "parameters": {
-                        "type": "object",
-                        "properties": {
-                            "age": {"type": "integer"}
-                        }
-                    }
-                }
-            }
-        ]
+        "functions": [record_favorite_color_func],
     }
+```
 
-def create_young_adult_node() -> Dict[str, Any]:
-    """Create node for young adult quotes."""
-    return {
-        "name": "young_adult",
-        "task_messages": [
-            {
-                "role": "system",
-                "content": "Explain our special young adult coverage options."
-            }
-        ],
-        "functions": [...]  # Additional quote-specific functions
-    }
+### FlowConfig (Static Flow)
 
-def create_standard_node() -> Dict[str, Any]:
-    """Create node for standard quotes."""
-    return {
-        "name": "standard",
-        "task_messages": [
-            {
-                "role": "system",
-                "content": "Present our standard coverage options."
-            }
-        ],
-        "functions": [...]  # Additional quote-specific functions
-    }
+The FlowConfig is a static JSON object that completely defines the entire flow. This follows the same rules as the NodeConfig, but is assembled as a single JSON object that defines the entire state of the program.
 
-# Initialize flow manager
-flow_manager = FlowManager(
-    task=task,
-    llm=llm,
-    context_aggregator=context_aggregator,
-)
+The FlowConfig can contain functions that are used to define the function handler behavior that executes when a function call occurs.
 
-@transport.event_handler("on_first_participant_joined")
-async def on_first_participant_joined(transport, participant):
-    await transport.capture_participant_transcription(participant["id"])
-    await flow_manager.initialize(create_initial_node())
-```
+See the [food_ordering example](https://github.com/pipecat-ai/pipecat-flows/blob/main/examples/static/food_ordering.py) for an example of the FlowConfig.
 
-### Best Practices
+### Function Calls
 
-- Store shared data in flow_manager.state
-- Create separate functions for node creation
+Pipecat Flows supports three ways to define function calls:
 
-# Flow Editor
+#### FlowsFunctionSchema (Recommended)
```

Comment:
As I've been reading this doc and looking at our examples, I'm wondering if/when we should flip to recommending direct functions as the main recommended way of doing things, with function schemas as an alternative if you happen to need more fine-grained control (mainly over the `properties` object). Using functions directly really does cut down boilerplate and opportunities for making mistakes...